osu.Game/Storyboards/Drawables/DrawableStoryboardSample.cs

```diff
@@ -38,7 +38,7 @@ protected override void SkinChanged(ISkinSource skin, bool allowFallback)
             foreach (var mod in mods.Value.OfType<IApplicableToSample>())
             {
                 foreach (var sample in SamplesContainer)
-                    mod.ApplyToSample(sample);
+                    mod.ApplyToSample(sample.Sample);
```

Comment:
seems slightly suspect as `Sample` is annotated as `CanBeNull` (can this realistically fail in a gameplay scenario? if not, should have assertions maybe?)